apk-generator/v2/app/generator/generator.py

```diff
@@ -327,6 +329,9 @@ def run_command(self, command):
         rc = process.poll()
         return rc
 
+    def handle_message(self, message):
+        socketio.emit('message', message)
```

Comment:
Docs clearly say emit is for named events. We're just listening to the unnamed `message` event. So this should be `send` not `emit`. 

And what's the use of broadcasting to all clients ? I don't want to get the status updates of the app that you are generating (Given that both of us are using the generator at the same time). So, use namespaces and broadcast the status to the correct namespace so that only the clients connected to that specific namespace get the messages